docs/topics/modules/glossary.adoc

```diff
@@ -59,11 +63,7 @@ iteration:: An iteration is used to organize, plan, and execute work items in a
 // endterm
 
 // term: f05a151a-61fa-45b1-8d8b-b3fd7bc63ea9, en_EN
-license conflict:: Licenses that conflict at the stack or component level.
-// endterm
-
-// term: 3a953b07-0cc3-4b45-b891-bf490216eae3, en_EN
```

Comment:
You're removing a term (and thus an existing UUID) -- we cannot do that unless we have a confirmation from the Planner team that the UUID/term isn't consumed by them.

Reply:
As per our discussion on this, I have used this UUID for the term Restrictive license. As per latest Analytics UX, License outlier is replaced with Restrictive license. Planner team will not consume this as it is part of Analytics UX.